dbbuild/lib/marker.py

```diff
@@ -36,7 +36,9 @@ class Marker:
     def from_csv(csvpath, index, source=None):
         table = pd.read_csv(csvpath)
         for n, row in table.iterrows():
-            rsids = row.VarRef.split(";")
+            rsids = []
+            if not pd.isna(row.VarRef):
+                rsids = row.VarRef.split(";")
```

Comment:
This piece of dbbuild code was failing with the new data. It successfully handled marker definitions where 1 or more RSIDs were missing, e.g. for rare variants lacking an RSID. The code was updated in this PR to correctly handle marker definitions providing 0 RSIDs.